### DISCUSSION TITLE
Function Graph layouts

### DISCUSSION THREAD

**Discussion by sar5430** [NationalSecurityAgency/ghidra#2947] [Q&A]
Hello,

I'm wondering if it's somehow possible to use the different rendering layouts from the Graph view (especially the "Compact Hierarchical one) in the Function Graph ? I may be wrong but it seems that the rendering is done with Jung for both ? 

Thanks

**Comment by dragonmacher** [comment]
It is not yet something supported by the Function Graph API.  It should be fairly straight forward to do and is likely to happen in the future.    They both use Jung in some form, but use different versions of that API.

**Comment by sar5430** [comment]
Great, thanks for the information.
